shared/chat/conversation/normal/container.tsx

```diff
@@ -38,14 +38,39 @@ const useOrangeLine = () => {
     needRPC = true
   }
 
-  // if we're not active and new messages came in, get the orange line
+  // desktop if we're not active and new messages came in, get the orange line
   const active = C.useActiveState(s => s.active)
   if (!active) {
     if (maxVisibleMsgID > lastVisibleMsgIDRef.current) {
+      logger.info('[useOrangeLine debug] active with new messages detected')
       lastVisibleMsgIDRef.current = maxVisibleMsgID
       needRPC = true
     }
   }
+  // mobile if we background, clear the orange line
+  const mobileAppState = C.useConfigState(s => s.mobileAppState)
```

Comment:
when you're backgrounded just clear the orange line so we can get a new one on foreground if needed thats accurate